OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -808,7 +811,15 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 					self.CancelActivity();
 
 				self.SetTargetLine(Target.FromCell(self.World, cell), Color.Green);
-				self.QueueActivity(order.Queued, WrapMove(new Move(self, cell, WDist.FromCells(8), null, true)));
+
+				var selectionCount = self.World.Selection.ActorCount;
```

Comment:
`World.Selection` can't be used from synced code!

Reply:
#16883 adds plumbing to adress this, so I'll wait with rebasing & updating until that PR is merged.

Reply:
Actually, if you only need the selection count and not the selection itself it is not necessary to wait. You can already send a single integer with an order using the `ExtraData` field.

Reply:
Ah, right, that's probably the better approach anyway.